CONTRIBUTING.md

```diff
@@ -47,6 +47,31 @@ This project is built from several repositories:
 * [Percona-Lab/clickhouse_exporter](https://github.com/Percona-Lab/clickhouse_exporter)
 * [percona/percona-toolkit](https://github.com/percona/percona-toolkit)
 
+
+### PMM DBaaS
+
+#### Prerequisites
+
+1. Installed minikube
+1. Installed docker
+
+#### Running minikube
+
+To spin-up k8s cluster, run
+```
+    minikube start --cpus=4 --memory=7G --apiserver-names host.docker.internal --kubernetes-version=v1.23.0
+    ENABLE_DBAAS=1 NETWORK=minikube make env-up # Run PMM with DBaaS feature enabled
+```
+
+[Read the documentation](https://docs.percona.com/percona-monitoring-and-management/setting-up/server/dbaas.html) how to run DBaaS on GKE or EKS
+
+##### Troubleshooting
+
+1. You can face with pod failing with `Init:CrashLoopBackOff` issue. Once you get logs by running `kubectl logs pxc-cluster-pxc-0 -c pxc-init` you get the error `install: cannot create regular file '/var/lib/mysql/pxc-entrypoint.sh': Permission denied`. You can fix it using [this solution](https://github.com/kubernetes/minikube/issues/12360#issuecomment-1123794143). Also, check [this issue](https://jira.percona.com/browse/K8SPXC-879)
+1. Multinode PXC Cluster can't be created on ARM CPUs. You can have single node installation.
+1. Operators are not supported. This issue can happen in two different scenarios. You can have PMM version higher then current release, our you installed higher version of operators. You can check compatibility using https://check.percona.com/versions/v1/pmm-server/PMM-version
```

Comment:
all items have the number 1.

Reply:
Yep. But it renders correctly by github.

Reply:
But not locally :)